core/controllers/suggestion.py

```diff
@@ -344,8 +352,24 @@ def get(self, target_type, suggestion_type):
         """
         self._require_valid_suggestion_and_target_types(
             target_type, suggestion_type)
+        topic_name = self.request.get('topic_name', None)
+
+        opportunity_summary_exp_ids_specific_to_topic = []
+        if topic_name is not None:
+            topic = topic_fetchers.get_topic_by_name(topic_name)
+            if topic is not None:
+                exploration_opportunity_summaries = (
+                    opportunity_services.
+                    get_exploration_opportunity_summaries_by_topic_id(
+                    topic.id))
```

Comment:
Indent by 4.

Reply:
In a few other places, I saw that adding such indentation leads to pylint error of line being too long. So, I coded it in a way that it doesn't lead to PyLint errors. But I can shift this.

Reply:
Ack. FYI, the general rule here is that a new "scope" is being introduced with the '(', hence the additional indentation.